skills/src/main/java/cm/aptoide/skills/SkillsFragment.kt

```diff
@@ -57,15 +59,23 @@ class SkillsFragment : DaggerFragment() {
       binding.findOpponentButton.setOnClickListener {
         val userName = binding.userNameTv.text.toString()
         saveUserName(userName)
-        disposable.add(viewModel.getRoom(userId, userName, packageName, this)
-            .observeOn(AndroidSchedulers.mainThread())
-            .doOnSubscribe({ showLoading(R.string.finding_room) })
-            .doOnNext({ userData ->
-              requireActivity().setResult(RESULT_OK, buildDataIntent(userData))
-              requireActivity().finish()
-            })
-            .doOnNext { ticket -> println("ticket: " + ticket) }
-            .subscribe())
+        KeyboardUtils.hideKeyboard(view)
+
+        disposable.add(
+            handleWalletCreationIfNeeded()
+                .takeUntil { it != "CREATING" }
```

Comment:
Should use a constant instead of an hardcoded string.

Reply:
agreed, changed